packages/frontend/src/components/AccountListSidebar/AccountItem.tsx

```diff
@@ -139,7 +139,7 @@ export default function AccountItem({
         // set Timeout forces it to be run after react update
         setTimeout(() => {
           ActionEmitter.emitAction(KeybindAction.Settings_Open)
-        }, 0)
+        }, 100)
```

Comment:
I don't like adding delays to fix issues.

Reply:
settings should either use react suspense to wait for the settings store, or probably even better we should explore to remove the settings store https://github.com/deltachat/deltachat-desktop/issues/4200

Reply:
maybe too much work for the moment, to get the release out I'd approve it when there is also a comment here explaining the delay.